db/schema.rb

```diff
@@ -44,14 +44,15 @@
   end
 
   create_table "language_skills", force: :cascade do |t|
-    t.bigint "client_id"
+    t.bigint "languageable_id"
     t.string "language"
     t.string "level"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.datetime "deleted_at"
-    t.index ["client_id"], name: "index_language_skills_on_client_id"
+    t.string "languageable_type"
     t.index ["deleted_at"], name: "index_language_skills_on_deleted_at"
+    t.index ["languageable_id"], name: "index_language_skills_on_languageable_id"
```

Comment:
It is a migration to change, the methods used are different the have usually a syntax of:
```ruby
<add/remove/change>_<what> :<table_name>, :<attributes>
```
The `t. references` is only available when you write it with a block and I didn't saw a method which you can change a field to a polymorphic association